src/components/Card.jsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Stack, Typography } from '@mui/material';
+import { useState } from 'react';
+
+const Card = () => {
+  let [addToCart, setAddToCart] = useState(false);
+  let [value, setValue] = useState(1);
+
+  const Increment = () => setValue(value++); // increment value by 1
+
+  const Decrement = () => {
+    if (value >= 1) {
```

Comment:
This `if condition` has no effect and `setValue(value)` would be executed anyway as there is no early exit or `else condition`.
```
const handleDecrement = () => {
  if (value > 0) {
    setValue(value--);
  }
}
```

or 

```
const handleDecrement = () => {
  if (value <= 1) {
    return;
  }
  
  setValue(value);
}
```